src/pages/billing/CreateAzureBillingProjectModal.js

```diff
@@ -125,7 +126,13 @@ const CreateAzureBillingProjectModal = ({ onSuccess, onDismiss, billingProjectNa
         })
       ])]),
       h(IdContainer, [id => h(Fragment, [
-        h(FormLabel, { htmlFor: id, required: true }, ['Managed application']),
+        h(FormLabel, { htmlFor: id, required: true }, [
+          'Unassigned managed application',
+          h(InfoBox, { style: { marginLeft: '0.25rem' } }, [
```

Comment:
Should I add a unit test that asserts that this tooltip is rendered? Wasn't sure if it was necessary but happy to do so

Reply:
I don't think it's necessary.